index.html

```diff
@@ -19,8 +19,8 @@ <h2>Thank you for taking part.</h2>
       <label id="emiail-label"  for="email">Email</label>
       <input type="text" placeholder="Enter your email">
       <br>
-      <label id="age-label" for="number" >Age(optinal)</label><br>
-      <input type="number" min="10" max="99" placeholder="Age">
+      <label id="age-label" for="number" >Age</label><br>
+      <input type="number" min="10" max="99">
```

Comment:
Please, add a placeholder for the word "Age".
This will make the word 'Age' appear on your form which currently it's not showing on your form. 
Thanks 🙂